pages/service-usages/basic-usages.md

```diff
@@ -0,0 +1,209 @@
+```
+
+#### System Query Option $top and $skip
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People?$top=2 
+```
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People?$skip=18
+```
+
+#### System Query Option $count
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People/$count
+```
+
+#### System Query Option $expand
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('keithpinckney')?$expand=Friends
+```
+
+#### System Query Option $select
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports?$select=Name, IcaoCode
+```
+
+#### Lambda Operators
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Me/Friends?$filter=Friends/any(f:f/FirstName eq 'Scott')
+```
+
+### Data Modification
+
+#### Create an Entity
+
+```
+POST http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People
+header:
+{
+	Content-Type: application/json
+}
+body:
+{
+    "UserName":"lewisblack",
+    "FirstName":"Lewis",
+    "LastName":"Black",
+    "Emails":[
+        "lewisblack@example.com"
+    ],
+    "AddressInfo": [
+    {
+      "Address": "187 Suffolk Ln.",
+      "City": {
+        "Name": "Boise",
+        "CountryRegion": "United States",
+        "Region": "ID"
+      }
+    }
+    ]
+}
+```
+
+#### Remove an Entity
+
+```
+DELETE http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')
+```
+
+#### Update an Entity
+
+```
+PATCH http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')
+header:
+{
+	Content-Type: application/json
+}
+body:
+{
+    "@odata.type":"Microsoft.OData.Service.Sample.TrippinInMemory.Models.Person",
+    "FirstName": "Mirs",
+    "LastName": "King"
+}
+```
+
+### Functions and Actions
+
+#### Invoking Unbound Functions
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/GetNearestAirport(lat = 33, lon = -118)
+```
+
+#### Invoking bound Functions
+
+```
+Get http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')/Trips(0)/Microsoft.OData.Service.Sample.TrippinInMemory.Models.GetInvolvedPeople
+```
+
+#### Invoking Unbound Actions
+
+```
+POST http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/ResetDataSource
+```
+
+#### Invoking Bound Actions
+
+```
+POST http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')/Microsoft.OData.Service.Sample.TrippinInMemory.Models.ShareTrip
+header:
+{
+	Content-Type: application/json
+}
+body:
+{
+    "userName": "scottketchum",
+    "tripId": 0
+}
+```
+
+### ETag
+
+##### Update Entity with ETag
```

Comment:
Add first request, get etag, this is common user scenario, get data first and get etag.
And for etag, indicates two part, one in body and one in header.